src/client/store/assessment/slice.ts

```diff
@@ -11,13 +9,19 @@ export const assessmentSlice = createSlice<AssessmentState, SliceCaseReducers<As
   reducers: {},
   extraReducers: (builder) => {
     builder.addCase(initApp.fulfilled, (state, { payload }) => {
-      const { assessment, countries, regionGroups } = payload
+      const { assessment } = payload
 
       state.assessment = assessment
```

Comment:
should we remove countries and regionGroups from initApp if we don't use them here?